archiver.go

```diff
@@ -497,6 +529,84 @@ SELECT rec.visibility, row_to_json(rec) FROM (
 	ORDER BY created_on ASC, id ASC) rec; 
 `
 
+const lookupPurgedBroadcasts = `
+SELECT row_to_json(rec) FROM (
+	SELECT
+	NULL AS id,
+	mb.id AS broadcast,
+	jsonb_build_object('uuid', c.uuid, 'name', c.name) as contact,
+	COALESCE(mb.text->c.language, mb.text->$2, mb.text->'base') AS text,
+	NULL AS urn,
+	NULL AS channel,
+	'out' AS direction,
+	'flow' AS type,
+	'wired' AS status,
+	'visible' AS visibility,
+	'[]'::jsonb AS attachments,
+	'[]'::jsonb AS labels,
+	mb.created_on AS created_on,
+	mb.created_on AS sent_on
+	FROM msgs_broadcast_recipients br
+	JOIN LATERAL (select uuid, name, language FROM contacts_contact cc WHERE cc.id = br.contact_id AND cc.is_test = FALSE) AS c ON TRUE
+	LEFT JOIN msgs_broadcast mb ON br.broadcast_id = mb.id
```

Comment:
Why do we use `left join` here ?  broadcast_id on msgs_broadcast_recipients cannot be null